packages/create-ponder/src/index.ts

```diff
@@ -259,7 +263,22 @@ export async function run({
   }
 
   let subgraph: string | undefined = options.subgraph;
+  let subgraphProvider: SubgraphProviderId | undefined =
+    options.subgraphProvider;
   if (templateMeta.id === "subgraph") {
+    if (subgraphProvider === undefined) {
+      const result = await prompts({
```

Comment:
If the user doesn't include the `--subgraph-provider` option, they will get a prompt to select one:

![Uploading Screen Shot 2024-07-22 at 3.40.17 PM.png…]()

Reply:
Oh sweet, I initially tried this approach, but the prompt was never triggering for me.